Formula/yb-voyager.rb

```diff
@@ -0,0 +1,22 @@
+9class YbVoyager < Formula
```

Comment:
Extra character `9`!

Reply:
Yes sorry for that. Will be fixed in the next commit.